e2e/cypress/support/commands/form.js

```diff
@@ -26,11 +28,14 @@ Cypress.Commands.add(
     return cy
       .waitForRoute(apiRoute)
       .then((xhr) => {
-        const recordId = xhr.response.body.pid_value;
+        const recordId = xhr.response.body.pid_value || xhr.response.body.control_number;
 
-        if (recordId) {
+        if (recordId && !REDIRECT_TO_EDITOR.includes(collection)) {
```

Comment:
are you sure about the logic? even if there is `pid_value` it might go to the `else if` because of the `ReDRECT_TO_EDITOR` collection